### PR TITLE
[FSSDK-9621] remove duplicate reactNativeAsyncStorageCache

### PR DESCRIPTION
**Pull request by raju-opti**
## Summary
- There were multiple implementations of reactNativeAsyncStorageCache which was doing the same thing.

## Test plan
- refactored tests accordingly

## Issues
- FSSDK-9621

### CODE REVIEW

File: lib/modules/event_processor/reactNativeEventsStore.ts [hunk 1]
```diff
@@ -38,10 +38,11 @@ export class ReactNativeEventsStore<T> {
 
   public async set(key: string, event: T): Promise<string> {
     await this.synchronizer.getLock()
-    const eventsMap: {[key: string]: T} = await this.cache.get(this.storeKey) || {}    
+    const eventsMap: {[key: string]: T} = await this.getEventsMap();
+    // const eventsMap: {[key: string]: T} = await this.cache.get(this.storeKey) || {}
```
Comment (mikechu-optimizely) [inline lib/modules/event_processor/reactNativeEventsStore.ts:42]: NIT cool if we remove this contented code?

File: lib/modules/datafile-manager/datafileManager.ts
Comment (mikechu-optimizely) [inline lib/modules/datafile-manager/datafileManager.ts:1]: Can you update the copyright on each file?

**Review by mikechu-optimizely — CHANGES_REQUESTED** [review]
Thanks for simplifying. Just small things below.

File: lib/modules/event_processor/reactNativeEventsStore.ts (hunk 1 above)
Comment (raju-opti) [reply lib/modules/event_processor/reactNativeEventsStore.ts]: updated

**Pull request merged**